src/Modules/Payments/Application/PriceListItems/CreatePriceListItem/CreatePriceListItemCommand.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using CompanyName.MyMeetings.Modules.Payments.Application.Configuration.Commands;
+using Newtonsoft.Json;
+
+namespace CompanyName.MyMeetings.Modules.Payments.Application.PriceListItems.CreatePriceListItem
+{
+    public class CreatePriceListItemCommand : InternalCommandBase<Guid>
+    {
+        public string CountryCode { get; }
+
+        public string SubscriptionPeriodCode { get; }
+        
+        public string CategoryCode { get; }
+        
+        public decimal PriceValue { get; }
+
+        public string PriceCurrency { get; }
+
+        [JsonConstructor]
```

Comment:
To remove, only needed for internal commands

Reply:
Fixed